benchmarks/test_hmf.py

```diff
@@ -0,0 +1,96 @@
+import os
+import numpy as np
+import pyccl as ccl
+
+# Set cosmology
+cosmo = ccl.Cosmology(Omega_c=0.25, Omega_b=0.05, Omega_g=0, Omega_k=0,
+                      h=0.7, sigma8=0.8, n_s=0.96, Neff=0, m_nu=0.0,
+                      w0=-1, wa=0, T_CMB=2.7255,
+                      transfer_function='eisenstein_hu')
+# Read data
+dirdat = os.path.dirname(__file__) + '/data/'
+
+# Redshifts
+zs = np.array([0., 0.5, 1.])
+
+
+def test_hmf_despali16():
+    hmd = ccl.halos.MassDef('vir', 'critical')
+    mf = ccl.halos.MassFuncDespali16(cosmo, hmd)
+    d_hmf = np.loadtxt(dirdat + 'hmf_despali16.txt', unpack=True)
```

Comment:
Here you want `os.path.join` too. I missed several of these in the other test file. All should be changed.